src/framework/theme/components/popover/helpers/adjustment.helper.ts

```diff
@@ -20,8 +23,12 @@ const NB_ORDERED_PLACEMENTS = {
   ],
 };
 
+@Injectable()
 export class NbAdjustmentHelper {
```

Comment:
I think we have to follow angular naming convention in this services and name them like **NbAdjustmentService**.